doc/quickstart.rst

```diff
@@ -0,0 +1,294 @@
+    - **Location** - a more exact location of the device in the building/room.
+    - **Status** - an asset's lifetime indicator. Newly bought assets has status "new". You can change it as required according to your own work flow.
+    - **Task url** - url to task in ticket system
+    - **Additional remarks** - additional info.
+    - **Service name** - service name to which the asset belongs
+    - **Property of** - to which the company belongs asset
+- **Financial Info**:
+    - **Price** - the unit price of the asset.
+    - **Provider** - the name of the provider of the asset.
+    - **Depreciation rate** - number of months this device deprecates
+    - **Source** - if this device was purchased or salvaged
+    - **Request date** - date of submission of the demand for the device.
+    - **Delivery date** - date of device delivery.
+    - **Deprecation end date** - the end day of the depreciation
+    - **Order number**, **Invoice date**, **Invoice no**, **Provider order date**, **Budget info**.
+- **User info**:
+    - **User** - device user.
+    - **Owner** - device owner.
+- **Aditional info**:
+    - **U level** - "U" level of installation device.
+    - **U height** - how large the device is, in "U".
+    - **Ralph device id** - ID device detected by Ralph Scan.
+
+
+Bulk Editing
+------------
+
+It is often required to edit multiple assets at once. For example, when you
+want to move them from one warehouse to another. There is a special mode called
+"bulk edit" for this case.
+
+To activate this mode, go to the search screen, and select multiple assets
+using check marks on the left side.
+
+.. image:: _static//bulk-1.png
+
+When ready, choose "Edit selected" from the bulk edit actions.
+
+.. image:: _static//bulk-2.png
+    :scale: 55%
+
+On the next screen you can edit those records all at once by changing the
+appropriate fields. When you fill one field with the desired value, you can
+propagate this value to all records by clicking on the "plus" mark near the
+current cell.
+
+
+Work Flow and Statues
+---------------------
+
+.. image:: _static/edit-device-status.png
+
+
+In this version there are no limits for moving assets from one status to
+another.  You can freely change statuses.  All changes will be recorded,
+allowing you to inspect the flow later.
+
+
+Licences
+^^^^^^^^
+Ralph Assets allows you to store information about software licenses.
+Adding and editing is performed in much the same way as in assets.
+
+Adding License
+--------------
+
+To add a license, click the "Add support" option from the top of submenu.
+
+.. image:: _static/add_licence.png
+    :scale: 75%
+
+
+
+Licence fields are split into 2 section: **Basic info** and **Financial info**.
+**Financial info** contains very important field, **Number of purchased items**.
+This field ability to store Multi-Seat licence.
+
+
+Relations
+---------
+
+Licenses may be related to the relationship with the user or device.
```

Comment:
erm? Maybe *Licences ,may have a relationship with a user or device* or *Licenses may be related to a user or device*